### DISCUSSION TITLE
boa noite preciso de ajuda

### DISCUSSION THREAD

**Discussion by Geremiasdiias** [community/community#34655] [Other Feature Feedback, Questions, & Ideas]
eu nao estou conseguindo fazer uma solicitação para ganhar benefícios sendo que eu coloquei tudo conforme pedi

**Comment by SilasRodrigues19** [comment]
@Geremiasdiias Que benefício? do GitHub Student Developer Pack? Se for, edita a discussão colocando o print de como está a etapa de verificação
